aspnetcore/security/authentication/identity.md

```diff
@@ -190,6 +191,10 @@ These dependencies are needed to use the Identity system in ASP.NET Core applica
 
 For additional information and guidance on migrating your existing Identity store see [Migrating Authentication and Identity](xref:migration/identity).
 
+## Setting password strength
+
+See [this section](#pw) for a sample that sets the minimum password requirements.
```

Comment:
[this section] change to something descriptive (like the name of the section [Configuration]) for SEO as well as accessibility. Accessible readers can't determine what's going on here for those who have vision issues.